src/components/crown/crownMultiselect/crownMultiselect.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <div
+    class="crown-config crown-multiselect"
+    :style="style"
+    v-if="isMultiSelect"
+    role="menu"
+  >
+    <slot />
+
+    <button
+      v-for="button in buttons"
+      :key="button.label"
+      :aria-label="button.label"
+      class="btn"
+      :title="button.label"
+      :data-test="button.testId"
+      :role="button.role"
+      @mousedown.stop.prevent
+      @click.stop.prevent="button.action"
+    >
+      <i :class="`fa fa-${button.icon} text-white`" />
+    </button>
+  </div>
+</template>
+
+<script>
+import store from '@/store';
+import runningInCypressTest from '@/runningInCypressTest';
+
+export default {
+  props: {
+    paper: Object,
+  },
+  data() {
+    return {
+      runningInCypressTest: runningInCypressTest(),
+      showCrown: false,
+      savePositionOnPointerupEventSet: false,
+      style: null,
+      taskDropdownInitiallyOpen: this.paperNotRendered(),
+      showReplaceModal: false,
+      nodeToReplace: null,
+      buttons: [
+        {
+          label: 'Copy Element',
+          icon: 'copy',
+          testId: 'copy-button',
+          role: 'menuitem',
+          action: this.copyElement,
+        },
+        {
+          label: 'Delete Element',
+          icon: 'trash-alt',
+          testId: 'delete-button',
+          role: 'menuitem',
+          action: this.deleteElement,
+        },
+        // add more buttons as necessary
+      ],
+    };
+  },
+  created() {
+    this.$t = this.$t.bind(this);
+  },
+  computed: {
+    isMultiSelect() {
+      const countSelected = store.getters.highlightedShapes.length;
+      return countSelected > 1;
+    },
+    isFlow() {
+      return false;
+    },
+    highlightedShapes: () => store.getters.highlightedShapes,
+  },
+  methods: {
+    copyElement() {
+      // @todo: Implement copyElement
```

Comment:
remove the todo if completed

Reply:
This will be implemented in the copy/paste ticket.